pkgs/applications/audio/littlegptracker/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, fetchFromGitHub
+, SDL
+, jack2
+, Foundation
+}:
+
+stdenv.mkDerivation rec {
+  pname = "littlegptracker";
+  version = "unstable-2019-04-14";
+
+  src = fetchFromGitHub {
+    owner = "Mdashdotdashn";
+    repo = "littlegptracker";
+    rev = "0ed729b46739e3df5e111c6fa4d548fde2d3b891";
+    sha256 = "1pc6lg2qp6xh7ahs5d5pb63ms4h2dz7ryp3c7mci4g37gbwbsj5b";
+  };
+
+  buildInputs = [
+    SDL
+    jack2
+  ] ++ stdenv.lib.optional stdenv.isDarwin Foundation;
+
+  postPatch = ''
+    # Remove coherency checks >:)
+    # As far as I can see, they are just to make sure that the types can fit
+    # in the word length, but this check was written when 64 bits weren't a
+    # possibility and didn't take that into account,
+    # so although the types do fit, the checks fail.
+    # Indeed, the program seems to work well on 64 bits.
+    # More info here:
+    # https://github.com/Mdashdotdashn/LittleGPTracker/issues/4
+    sed -i '126,147d' sources/Externals/Soundfont/DATATYPE.H
```

Comment:
I'd prefer this to be a patch instead, so the build breaks more obviously when that file is updated.